functests/utils/nodes/nodes.go

```diff
@@ -201,3 +201,31 @@ func BannedCPUs(node corev1.Node) (banned cpuset.CPUSet, err error) {
 
 	return banned, nil
 }
+
+// GetDefaultSmpAffinitySet returns the default smp affinity mask for the node
+func GetDefaultSmpAffinitySet(node *corev1.Node) (cpuset.CPUSet, error) {
+	command := []string{"cat", "/proc/irq/default_smp_affinity"}
+	defaultSmpAffinity, err := ExecCommandOnNode(command, node)
+	if err != nil {
+		return cpuset.NewCPUSet(), err
+	}
+	defaultSmpAffinitySet, err := components.CPUMaskToCPUSet(defaultSmpAffinity)
```

Comment:
You can just return the output of components.CPUMaskToCPUSet

Reply:
ah, right! thanks